internal/fleetdb/fleetdb.go

```diff
@@ -40,36 +45,31 @@ func NewFleetDBClient(ctx context.Context, cfg *app.Configuration) (Client, erro
 
 	return &fleetDBClient{
 		client:                     client,
-		slugs:                      slugs,
 		inventoryConverterInstance: inventoryconverter.NewInventoryConverter(slugs),
 	}, nil
 }
 
 type fleetDBClient struct {
 	client                     *fleetdb.Client
-	slugs                      map[string]*fleetdb.ServerComponentType
 	inventoryConverterInstance *inventoryconverter.InventoryConverter
 }
 
-func (fc fleetDBClient) GetServer(ctx context.Context, id uuid.UUID) (*fleetdb.Server, *fleetdb.ServerResponse, error) {
-	return fc.client.Get(ctx, id)
+func (fc fleetDBClient) GetServerInventory(ctx context.Context, id uuid.UUID, inband bool) (*rivets.Server, *fleetdb.ServerResponse, error) {
+	return fc.client.GetServerInventory(ctx, id, inband)
 }
 
 func (fc fleetDBClient) GetComponents(ctx context.Context, id uuid.UUID, params *fleetdb.PaginationParams) (fleetdb.ServerComponentSlice, *fleetdb.ServerResponse, error) {
 	return fc.client.GetComponents(ctx, id, params)
```

Comment:
I'm ambivalent about keeping this call. But let's not change anything in the PR for it.